packages/@okta/vuepress-theme-prose/layouts/Layout.vue

```diff
@@ -213,11 +213,6 @@ export default {
     this.$on("toggle-tree-nav", event => {
       that.appContext.isTreeNavMobileOpen = event.treeNavOpen;
     });
-    
-    // Delay height adjustment on mount to ensure the header element is fully rendered.
-    setTimeout(() => {
```

Comment:
Moved this section to `HeaderBanner` to ensure the headerHeight is re-calculated only when the banner is present.